web/src/components/common/components/calendar/calendar.tsx

```diff
@@ -26,6 +26,17 @@ const Calendar: React.FC<Content<'content.calendar'>> = ({ calendarUrl }) => {
     window.open(`/api/calendar/download?c=${calendarUrl}`, '_blank');
   };
 
+  // Function to update localstorage whenever the includePast variable is set
+  const updateIncludePast = (value: boolean) => {
+    localStorage.setItem("includePast", String(value))
+    setIncludePast(value)
+  }
+
+  // Triggers on page load to update the includePast variable to the value last set by the user
+  useEffect(() => {
+    setIncludePast(window.localStorage.getItem("includePast")==="true")
```

Comment:
**Suggestion**
We can load this variable directly into the useState at the top instead of waiting for the useeffect to trigger

```suggestion
    useState(window.localStorage.getItem("includePast")==="true")
```

Also, just a nitpick, but add some spaces around the `===` for readability :)